src/pretalx/api/serializers/room.py

```diff
@@ -23,7 +23,15 @@ def get_url(self, obj):
 
     class Meta:
         model = Room
-        fields = ("id", "guid", "name", "description", "capacity", "position", "url")
+        fields = (
+            "id",
+            "uuid",
```

Comment:
This changes the REST API to provide a "uuid" attribute rather than a "guid" attribute. That's a breaking change, which doesn't seem necessary, right?

Reply:
oh, it totally missed that... what is the proper way to export the value of the uuid property with the guid key?

Reply:
Might work with a `UUIDField(source='uuid')`, but I'm not sure if `source` doesn't maybe check for actual proper fields. In that case, a `SerializerMethodField()`, and `get_guid(self, instance): return instance.uuid`, -ish, off the top of my head.